app/serializers/user_serializer.rb

```diff
@@ -58,4 +66,9 @@ def show_firebase_token?
   def add_links(model, data)
     data[:links] = {}
   end
+
+  def permitted_value(value)
+    permitted_requester? ? value : ""
```

Comment:
With regards to #720, can set a different value depending on the attribute's type. I think we said 0 was okay for Integer fields as a 'blank' value.